package.json

```diff
@@ -180,6 +188,7 @@
     "rxjs": "^6.0.0",
     "rxjs-marbles": "^6.0.0",
     "vanilla-picker": "^2.10.1",
-    "wavesurfer.js": "^5.2.0"
+    "wavesurfer.js": "^5.2.0",
+    "wdio-video-reporter": "^3.2.1"
```

Comment:
Why is this added to `dependencies` and not `devDependencies`?

Reply:
It was added here by mistake, now added in devDependencies.